app/plugin.go

```diff
@@ -394,3 +398,48 @@ func (a *App) GetPlugins() (*model.PluginsResponse, *model.AppError) {
 
 	return resp, nil
 }
+
+// notifyPluginEnabled notifies web clients for all cluster peers if
+// manifest version is same across all peers, including self.
+func (a *App) notifyPluginEnabled(manifest *model.Manifest) *model.AppError {
+	if manifest.HasClient() {
+		var statuses model.PluginStatuses
+		expectedPeers := 1
```

Comment:
nit: I'd expect `expectedPeers := 0`, since intuitively I am not my own `peer`.

Reply:
n/a. since flipping the logic to not notify when peers are not consistent.